README.md

```diff
@@ -1,182 +1,134 @@
-  open(path: string, options: any): JSVirtualFile;
-}
-
-export class JSVirtualFile {
-  lastAccessed(): BigInt;
-  lastModified(): BigInt;
-  createdTime(): BigInt;
-  size(): BigInt;
-  setLength(new_size: BigInt): void;
-  read(): Uint8Array;
-  readString(): string;
-  write(buf: Uint8Array): number;
-  writeString(buf: string): number;
-  flush(): void;
-  seek(position: number): number;
-}
-```
+          const { code, stdoutUtf8 } = await instance.wait();
 
-## Building
+          console.log(`Python exited with ${code}: ${stdoutUtf8}`);
+      }
 
-To build this library you will need to have installed in your system:
+      runPython();
+</script>
+```
 
-* Node.JS
-* [Rust][Rust]
-* [wasm-pack][wasm-pack]
-* [wabt][wabt] (for `wasm-strip`)
-* [binaryen][binaryen] (for `wasm-opt`)
+Alternatively, the package can be imported directly from the CDN by turning the
+script into a module.
 
-```sh
-npm i
-npm run build
-```
+```html
+<script defer type="module">
+    import { init, Wasmer } from "https://unpkg.com/@wasmer/sdk@latest/dist/Library.mjs";
 
-## Testing
+    async function runPython() {
+        await init();
 
-Build the pkg and run the tests:
+        ...
+    }
 
-```sh
-npm run build
-npm run test
+    runPython();
+</script>
 ```
 
-# What is WebAssembly?
+### Cross-Origin Isolation
 
-Quoting [the WebAssembly site](https://webassembly.org/):
+Browsers have implemented security measures to mitigate the Spectre and Meltdown
+vulnerabilities.
 
-> WebAssembly (abbreviated Wasm) is a binary instruction format for a
-> stack-based virtual machine. Wasm is designed as a portable target
-> for compilation of high-level languages like C/C++/Rust, enabling
-> deployment on the web for client and server applications.
+These measures restrict the sharing of `SharedArrayBuffer`` objects with Web
+Workers unless the execution context is deemed secure.
 
-About speed:
+The `@wasmer/sdk` package uses a threadpool built on Web Workers and requires
+sharing the same `SharedArrayBuffer` across multiple workers to enable WASIX
+threads to access the same address space. This requirement is crucial even for
+running single-threaded WASIX programs because the SDK internals rely on
+`SharedArrayBuffer` for communication with Web Workers.
 
-> WebAssembly aims to execute at native speed by taking advantage of
-> [common hardware
-> capabilities](https://webassembly.org/docs/portability/#assumptions-for-efficient-execution)
-> available on a wide range of platforms.
+To avoid Cross-Origin Isolation issues, make sure any web pages using
+`@wasmer/sdk` are served over HTTPS and have the following headers set:
 
-About safety:
+```yaml
+"Cross-Origin-Opener-Policy": "same-origin"
+"Cross-Origin-Embedder-Policy": "require-corp"
+```
 
-> WebAssembly describes a memory-safe, sandboxed [execution
-> environment](https://webassembly.org/docs/semantics/#linear-memory) […].
+See the [`SharedArrayBuffer` and Cross-Origin Isolation][coi-docs] section under
```

Comment:
This link doesn't render properly (see the rendered markdown in Github)

Reply:
Fixed.